zcash_primitives/src/transaction/tests/data.rs

```diff
@@ -137,6 +137,1358 @@ pub mod tx_read_write {
         0xb7, 0xe9, 0x4d, 0xb3, 0x90, 0xaa, 0x4b, 0xb7, 0x6b, 0x66, 0xa5, 0x34, 0xe5, 0xe2, 0x67,
         0x9b, 0x27, 0xdb, 0x5f, 0x95, 0xfd, 0x09, 0xa3, 0x6b, 0x05,
     ];
+
+    pub(crate) struct TestVector {
+        pub(crate) tx: Vec<u8>,
+        pub(crate) txid: [u8; 32],
+    }
+
+    // From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/zip_0244.py
+    pub(crate) fn v5_test_vectors() -> Vec<TestVector> {
+        vec![
+            TestVector {
+                tx: vec![
+                    0x05, 0x00, 0x00, 0x80, 0x0a, 0x27, 0xa7, 0x26, 0xb4, 0xd0, 0xd6, 0xc2, 0x7a,
```

Comment:
I checked this first line matched, which satisfied me that these were just copy-pasted.